english/commit-suicide/suicide.md

```diff
@@ -1,2 +1,5 @@
 Click [here](http://www.addictdepression.com/suicide-methods/) for some ideas.
-However, you must consider [these](http://lostallhope.com/things-consider) matters first.
\ No newline at end of file
+However, you must consider [these](http://lostallhope.com/things-consider) matters first.
+
+If you would like someone to talk to about your suicidal inclinations, 
+[here are some hotlines you can call.](https://www.reddit.com/r/SuicideWatch/wiki/hotlines)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.